x/stake/keeper/key.go

```diff
@@ -230,3 +230,11 @@ func GetREDsByDelToValDstIndexKey(delegatorAddr, validatorDstAddr sdk.AccAddress
 		GetREDsToValDstIndexKey(validatorDstAddr),
 		delegatorAddr.Bytes()...)
 }
+
+// get the prefix keyspace for all redelegations redelegating from a source validator
+// from a particular delegator
+func GetREDsByDelFromValSrcIndexKey(delegatorAddr, validatorSrcAddr sdk.AccAddress) []byte {
```

Comment:
Godoc should be of the form `GetREDsByDelFromValSrcIndexKey ...`. Also, the wording is a bit confusing I think.

Reply:
Yeah I was initially using that key but forgot to delete it from here